templates/ingress.yaml

```diff
@@ -31,7 +31,7 @@ spec:
     - host: {{ .host | quote }}
       http:
 	paths:
-	{{- range .paths }}
+          {{- range .paths }}
```

Comment:
The solution in the lab says:
>Opening the file and going to the appropriate line containing paths:, we notice that a tab instead of whitespace characters was used to indent.

So this should stay a tab.

Reply:
The solution only mentions the line containing `paths:` but not the following line. But yes, we could expect the attendant to fix both lines.

Reply:
Ah, I see, you are right. Let's keep your suggestion then :+1: